contact_updater/templates/agency_form.html

```diff
@@ -0,0 +1,42 @@
+{% extends "base.html" %}
+
+{% block body %}
+    <div class="hero">
+    <div class="container">
+    <section class="hero--content">
+
+
+    {% if validated  %}
+        <h3>Instructions</h3>
+    {% endif %}
+
+    <form action="" method="post">
+        {% csrf_token %}
+        {{management_form|safe}}
+        <fieldset>
+        {% if validated %}
+        <div style="display:none">
+        {% endif %}
+        {% for agency, form in data %}
+            <h2>{{agency.name}}</h2>
+            {{form|safe}}
+        {% endfor %}
+        <br>
+        </div>
```

Comment:
Does this closing tag need to be in a `if validated` block, too? Something to consider: it might make sense to render the form using only hidden inputs rather than rendering the whole form inside a `display:none`